package.json

```diff
@@ -1024,9 +1026,10 @@
     "nyc": "^15.1.0",
     "pinst": "^2.1.6",
     "rimraf": "^3.0.2",
+    "semantic-release": "^19.0.3",
```

Comment:
Are we going to go with `semantic-release`? I remember we discussed the possibility of using https://github.com/changesets/changesets, but I never got around to researching it. If you say `semantic-release` is great, then I suppose we could/should use it for everyhing?

Reply:
I stuck with semantic-release as it's what I know, I kinda like how with semantic-release I can just pull in the vsce plugin and it then handles all the publishing rather than having to do all that manually or in CI.

I still think changesets would totally be a good fit for the SDK though as it promotes better changelog entries than conventional-commits does

Reply:
I can second that! For smaller projects semantic-release is great because it automates a lot. However, in a bigger project changesets is really cool because you can add extensive changlog entries with each PR that are not tied to the commit message. Then, on release combine them all into one via CI and boom, nice and extensive changelog info. It has a lot of else to offer of course, that mostly applies to monorepos.